tests/test_canister_mo/src/test_canister_mo/main.mo

```diff
@@ -73,8 +73,8 @@ actor class TestCanister() {
   let params = IcWebSocketCdk.WsInitParams(
     handlers,
     null,
-    ?Nat64.fromNat(2_000),
-    ?Nat64.fromNat(1_000),
+    null, // ?Nat64.fromNat(2_000),
+    null, // ?Nat64.fromNat(1_000),
```

Comment:
> null, // ?Nat64.fromNat(2_000),
    null, // ?Nat64.fromNat(1_000),

why not setting them the same as in the rust test canister? https://github.com/omnia-network/ic-websocket-gateway/pull/11/files#diff-bb377c8907f3253c2b8808b6adc82f63b01e6c6cb6755fbf442f6505b8561199